### PR TITLE
Update development havener file

### PR DESCRIPTION
**Pull request by qu1queee**
Add support for ENV variables
Remove common prefix `chart` from releases list. While this format have been deprecated in the latest release.

This is a no-op, please merge.

### CODE REVIEW

**Comment by qu1queee** [comment]
@viovanov can we merge?

File: dev-env-havener.yaml [hunk 1]
```diff
@@ -29,16 +29,22 @@
...
-  chart_location: deploy/helm/cf-operator/
+- name: cf-operator
+  namespace: quarks
+  location: deploy/helm/cf-operator/
   overrides:
     image:
-      tag: (( shell . bin/include/versioning && echo "${VERSION_TAG}" )) 
-      repository: cf-operator
-      org: cfcontainerization
+      tag: (( env IMAGE_TAG ))
```
Comment (manno) [inline dev-env-havener.yaml]: we already have `DOCKER_IMAGE_TAG`, can we use that here too?

File: dev-env-havener.yaml [hunk 2]
```diff
@@ -29,16 +29,22 @@
...
+      repository: (( env DOCKER_REPOSITORY ))
+      org: (( env DOCKER_ORGANIZATION ))
     customResources:
-      enableInstallation: false
+      enableInstallation: (( env CRDS_ENABLED ))
+
+env:
+  IMAGE_TAG: (( shell . bin/include/versioning && echo "${VERSION_TAG}" ))
```
Comment (manno) [inline dev-env-havener.yaml]:
```suggestion
  DOCKER_IMAGE_TAG: (( shell . bin/include/versioning && echo "${VERSION_TAG}" ))
```

File: dev-env-havener.yaml (hunk 1 above)
Comment (qu1queee) [reply dev-env-havener.yaml]: yes

**Pull request merged**